tooling/os2mo_data_import/os2mo_data_import/mora_data_types.py

```diff
@@ -99,7 +103,8 @@ def build(self):
                 "uuid": self.type_ref_uuid
             }
         }
-
+        if self.visibility_ref:
```

Comment:
There is no need to include visibility if the uuid does not exist, 
would it not be better to change the conditional to "if self.visibility_ref_uuid:" ?